plugins/report_result.d/01_dmesg_check

```diff
@@ -34,7 +34,7 @@ FAILUREFILENM=${FAILUREFILENM:-"/usr/share/rhts/failurestrings"}
 if [ ! -s "${FAILUREFILENM}" ]; then
     if [ -z "${FAILURESTRINGS}" ]; then
         DMESG_FAIL_SELECTOR="Default FAILURESTRINGS"
-        FAILURESTRINGS="Oops|BUG|NMI appears to be stuck|Badness at"
+        FAILURESTRINGS="\bOops\b|\nBUG\b|NMI appears to be stuck|Badness at"
```

Comment:
it looks like a typo. `\nBUG\b` you want `\bBUG\b`

Reply:
@bgoncalv, thank you very much for catching the typo! Let me have a try again after fixing the typo.